src/api/app/components/add_review_dropdown_component.html.haml

```diff
@@ -0,0 +1,20 @@
+.dropdown#add-review-dropdown-component
+  .btn.btn-success.dropdown-toggle{ type: 'button', data: { toggle: 'dropdown' }, aria: { expanded: 'false' } }
+    Review
+  .dropdown-menu.dropdown-menu-right
+    - @my_open_reviews.each do |review|
+      = button_tag(type: 'button', class: 'dropdown-item', data: { toggle: 'collapse', target: '#review_form_collapse',
+                   review: review.id }, aria: { expanded: 'false', controls: 'review_form_collapse' }) do
+        - case
```

Comment:
I still think you also need `Add a review for` or `for` here otherwise the dropdown doesn't make much sense...

![Screenshot from 2022-08-12 16-49-35](https://user-images.githubusercontent.com/514785/184380935-4abbf5a9-553a-4c87-9ce5-5e0608c1b2bf.png)

Reply:
The click target feel also very unsafe on mobile, they should have more margin there.

Reply:
I added a heading to the dropdown